code/android-core-library/src/main/java/com/adobe/marketing/mobile/services/ui/UIFullScreenMessage.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
We already have `AEPMessage`, `FullScreenMessage` and `FullScreenMessageDelegate` interfaces in `services.ui` package.
Can you reuse those instead of defining new interfaces?

Reply:
removed the fullscreen message classes as target will use the existing fullscreen message class within core 2.0